Release Day: 2022.11.14 (#42)
bottlesdevs/website: _posts/2022-11-14-release-2022.11.14.md

---
title: "Release Day 2022.11.14"
description: "Today is release day!"
published: 2022-11-14
layout: post
thumb: https://raw.githubusercontent.com/bottlesdevs/data/main/releases/2022.10.14/release-day.png
background: https://raw.githubusercontent.com/bottlesdevs/data/main/backgrounds/2022.10.14.png
published: true
comments: true
is_release_day: true
is_release_day_color: '#1699e7'

> user_nick: mirkobrombin
> user_name: Mirko Brombin
> user_tag: Founder
> co_user_nick: TheEvilSkeleton
> co_user_name: Hari Rana (TheEvilSkeleton)
> co_user_tag: Developer

Reviewer: ```suggestion
user_nick: cyberphantom52
user_name: Inam Ul Haq (cyberphantom52)
user_tag: Contributor
co_user_nick: TheEvilSkeleton
co_user_name: Hari Rana (TheEvilSkeleton)
co_user_tag: Developer
```

---

> title: "Release Day 2022.11.14"

Reviewer: Having proper titles should help readers understand the theme of the update.

```suggestion
title: "Release Day 2022.11.14: Interface Overhaul"
```

description: "Today is release day!"
published: 2022-11-14
layout: post
thumb: https://raw.githubusercontent.com/bottlesdevs/data/main/releases/2022.10.14/release-day.png
background: https://raw.githubusercontent.com/bottlesdevs/data/main/backgrounds/2022.10.14.png
published: true
comments: true
is_release_day: true

> is_release_day_color: '#1699e7'

Reviewer: ```suggestion
is_release_day_color: '#FF5F1F'
```

> user_nick: mirkobrombin

Reviewer 2: not this time

Reviewer: No worries. I'm reviewing right now and adding corrections.

user_name: Mirko Brombin
user_tag: Founder
co_user_nick: TheEvilSkeleton
co_user_name: Hari Rana (TheEvilSkeleton)
co_user_tag: Developer
---

It’s Release Day again!

A lot of work has been done this time around and it’s time to celebrate another release day!

This release comes with a redesign of the details and settings page. Also the programs page has been removed.

## News
Ok now let’s get to the point. What’s new in this version?

### Brand new details view

> The details view has been completely redesigned to make the ui more mobile friendly, the sidebar is gone and the pages have been moved into the details view itself. 

Reviewer: ```suggestion
The details view has been completely redesigned to make Bottles more mobile friendly. The sidebar has been removed and the pages have been moved into the details view itself.
```

The option to run an executable is now a nice visible pill to bring it into the focus and launch options are now neatly placed in their own popover next to it. There is a new bottom bar below the programs list to add/install new programs.

<div class="grid-pics">

>     <img tooltip="Details View" class="on-light" src="/uploads/details-view-light.png" />

Reviewer: This image doesn't work. The file path is `/uploads/details-view.png` (but I suggest renaming to `details-view-light.png`).

Also, the image is out of focus for both light and dark styles.

>     <img tooltip="Details View" class="on-light" src="/uploads/details-view-launchopts.png" />

Reviewer: This image works, but I suggest renaming the image to `details-view-launchopts-light.png`.

<div class="grid-pics">
    <img tooltip="Details View (dark)" class="on-dark" src="/uploads/details-view-dark.png" />
</div>

### Redesigned settings

> Settings page has also recieved an overhaul, similar settings have been rearranged into groups making it easier to navigate through them.

Reviewer: ```suggestion
Settings page has also received an overhaul, similar settings have been rearranged into groups making it easier to navigate through them.
```

### Redesigned settings
Settings page has also received an overhaul, similar settings have been rearranged into groups making it easier to navigate through them.

<div class="grid-pics">

>     <img tooltip="Settings View" class="on-dark" src="/uploads/settings-view-light.png" />

Reviewer: ```suggestion
    <img tooltip="Settings View" class="on-light" src="/uploads/settings-view-light.png" />
```

<div class="grid-pics">
    <img tooltip="Settings View (dark)" class="on-dark" src="/uploads/settings-view-dark-p1.png" />
    <img tooltip="Settings View Pt.2 (dark)" class="on-dark" src="/uploads/settings-view-dark-p2.png" />
</div>

## All other changes
There are **many other changes and improvements** in this release. Below are 
some noteworthy ones, for all others, please refer to the 
[GitHub organization](https://github.com/bottlesdevs).

* Bottle details automatically show on bottle creation
* Bottles now complies with GNOME Circle guidelines, see [#2126](https://github.com/bottlesdevs/Bottles/issues/2126)

> * Disable pulseaudio_latency by deafault.

Reviewer: ```suggestion
* Disable pulseaudio_latency by default.
```

---
title: "Release Day 2022.11.14: Interface Overhaul"
description: "Today is release day!"
published: 2022-11-14
layout: post
thumb: https://raw.githubusercontent.com/bottlesdevs/data/main/releases/2022.10.14/release-day.png
background: https://raw.githubusercontent.com/bottlesdevs/data/main/backgrounds/2022.10.14.png
published: true
comments: true
is_release_day: true
is_release_day_color: '#FF5F1F'
user_nick: cyberphantom52
user_name: Inam Ul Haq (cyberphantom52)
user_tag: Contributor
co_user_nick: TheEvilSkeleton
co_user_name: Hari Rana (TheEvilSkeleton)
co_user_tag: Developer
---

It’s Release Day again!

A lot of work has been done this time around and it’s time to celebrate another release day!

This release comes with a redesign of the details and settings page. Also the programs page has been removed.

## News
Ok now let’s get to the point. What’s new in this version?

> 

Reviewer: ```suggestion

### Change of the release schedule
We are switching our release schedule from bi-weekly to monthly. We have dedicated all of our time to Bottles, but we have recently started to develop our own projects, like [Vanilla OS](https://vanillaos.org/), [Crystal Linux](https://getcryst.al/site), [Atoms](https://github.com/AtomsDevs/Atoms), [Upscaler](https://gitlab.com/TheEvilSkeleton/Upscaler) and others. In the meantime, we also dedicated our time to contribute to numerous projects.

Even with the schedule changes, we will continue to develop Bottles, fix bugs and much more. At the same time, we will have more time to polish between updates, as we did especially with this update.

Every 14th of the month we will release a new update, always with new features, bug fixes and more!

```

    <img tooltip="Details View" class="on-light" src="/uploads/details-view-launchopts-light.png" />
</div>

<div class="grid-pics">
    <img tooltip="Details View (dark)" class="on-dark" src="/uploads/details-view-dark.png" />
    <img tooltip="Details View (dark)" class="on-dark" src="/uploads/details-view-launchopts-dark.png" />
</div>

### Redesigned settings
Settings page has also received an overhaul, similar settings have been rearranged into groups making it easier to navigate through them.

<div class="grid-pics">
    <img tooltip="Settings View" class="on-light" src="/uploads/settings-view-light.png" />
</div>

<div class="grid-pics">
    <img tooltip="Settings View (dark)" class="on-dark" src="/uploads/settings-view-dark.png" />
</div>

### QoL Changes
A lot of quality of life improvements  have been made to Bottles to improve usability. "states" have been renamed to "snapshots", rephased dialogs that tell user to wait, renamed "Documentation" to "Help", added mnemonics, and a lot more. Changes to Bottles to comply with libadwaita design guidelines.

Refer to [#2228](https://github.com/bottlesdevs/Bottles/pull/2228) for full details.

> 

Reviewer: ```suggestion

### Removal of color lookup table
We decided to remove the color lookup table feature from the vkBasalt dialog. We realized that supporting color lookup tables is a really complicated task, and it's a topic we know very little about. We aren't sure if it's working correctly or not. We rather remove this feature than provide a potential broken feature. We apologize for users who have been using this feature, but if you understand how vkBasalt's color lookup table works, please don't hesitate to contact us!
